### ISSUE TITLE
Feature request: An example could be made with bracket-lib

### ISSUE THREAD

**Issue by erlend-sh** [xenith-studios/ataxia#3]
Are you familiar with [bracket-lib](https://github.com/thebracket/bracket-lib) and its accompanying Roguelike tutorial (of 70+ chapters)? It seems like a suitably minimalist game library for ataxia to demo a MUD on.

**Comment by xenith** [comment]
Thanks for the link! I haven't seen that one yet. I was planning to work through a roguelike tutorial, and this one looks rather good. I'm not sure what you envision for how it could work with ataxia though, could you elaborate a little bit?

I do want to note that the multiplayer aspect of ataxia is going to be designed more like a classic MUD, but there isn't much code written as of yet.

**Comment by erlend-sh** [comment]
> the multiplayer aspect of ataxia is going to be designed more like a classic MUD

Could you give an example of such a classic MUD?

My impression of MUD games is that they are predominantly text-based. As such, bracket-lib’s terminal emulator should be able to provide most of what you need in a first take on graphics, and it’d even work in the browser.

https://github.com/thebracket/bracket-lib/tree/master/bracket-terminal

**Comment by xenith** [comment]
A few examples would be the ROM, coffeemud, and tbamud codebases. For a live mud that I run, you could look at http://www.ageoflegacy.com. Another one is https://federation2.com.

Traditionally they all work via telnet, but I'm planning to use web sockets with a browser client (though I'm building telnet to start for testing and development), so something like bracket-lib could definitely work there.

**Comment by Twoshrubs** [comment]
I totally agree with the OP. I would say it would be client side rather than server side.

This is something I'm currently looking at, building a custom telnet mud client using rltk.. so your able to keep text look and feel of a MUD that we all love but also adding media which will hopefully open MUD's upto a newer player base (as were all getting older). 

I just need to find a server that I'm happy with.. that's why I'm here ;)
